contracts/components/staking/SubjectTypeValidator.sol

```diff
@@ -0,0 +1,85 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
We need to delete the file `SubjectTypes.sol` which apparently was moved to this one but got left over in some merge.